etna/metrics/functional_metrics.py

```diff
@@ -355,8 +375,24 @@ def wape(y_true: ArrayLike, y_pred: ArrayLike, multioutput: str = "joint") -> Ar
         raise ValueError("Shapes of the labels must be the same")
 
     axis = _get_axis_by_multioutput(multioutput)
-
-    return np.sum(np.abs(y_true_array - y_pred_array), axis=axis) / np.sum(np.abs(y_true_array), axis=axis)  # type: ignore
+    numerator = np.nansum(np.abs(y_true_array - y_pred_array), axis=axis)
```

Comment:
According to the documentation for versions of numpy > 1.9 `nansum` will return 0 for all empty array.  We should set it to `NaN` in such scenarios.
Seems like we don't have a test case when this behavior shows up (numerator computed from a full empty array and denominator has values).